supporting_scripts/analysis-of-endpoint-connections/src/main/typeScript/tsconfig.analysisOfEndpointConnections.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
+        "module": "CommonJS",
+        "target": "es2022",
```

Comment:
In the .ts file you have es2015. Is that mismatch on purpose?

Reply:
no. I updated the .ts file